exchanges/order/limits_test.go

```diff
@@ -290,7 +290,7 @@ func TestConformToAmount(t *testing.T) {
 
 	tt = MinMaxLevel{}
 	val := tt.ConformToAmount(1)
-	assert.Equal(t, 1.0, val) // If there is no step amount set this should not change
+	assert.Equal(t, 1.0, val) // If there is no step amount set this should not change the inputted amount
```

Comment:
🐒 Generally, test comments are actually test descriptions
```suggestion
	assert.Equal(t, 1.0, val, "ConformToAmount should return the same value with no step amount set")
```